package.json

```
@@ -5,8 +5,9 @@
   "description": "Draft for TC39 website",
   "main": "index.js",
   "scripts": {
+    "format:js": "npm run lint:js -- --fix",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "prettier -c --loglevel warn **/*.js",
+    "lint:js": "eslint --ext=.js,.mjs .",
```

```suggestion
    "lint:js": "eslint --ext=.js,.json,.mjs .",
```

Do I add the `.json` extension here? ESLint is already set up to lint JSON, but I'm unsure if this should be its own package run script or if this should be included in this line.

---

If that is no desirable, I will rebase to remove https://github.com/tc39/tc39.github.io/pull/213/commits/9110a57a4303c4437150e51b54920d6f03712663, let me know.

---

Seems reasonable to me to include it, if it can be linted :-) altho any attempt to consume it would fail if invalid, so i'm not sure it's useful.

---

I was kinda hoping that this would notify about improper indentation (e.g., using tabs instead of spaces or not indenting at all), but it doesn't seem to be working configured this way. Any idea what I've done wrong?

---

Oh dang, I just read https://github.com/prettier/eslint-plugin-prettier/issues/294 and it doesn't seem possible to do it this way. :(